### PR TITLE
[PE-4938] Improved drawer doc examples

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/PE-4938

Added functional examples for Web components and Html Blueprint code snippets in Drawer component page.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://fd99030cc9077842d614193d352316e4ef81dada-chi-pr.enterprise.ctl.io" target="_blank"> https://fd99030cc9077842d614193d352316e4ef81dada-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/988/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://fd99030cc9077842d614193d352316e4ef81dada-chi-pr.enterprise.ctl.io" target="_blank"> https://fd99030cc9077842d614193d352316e4ef81dada-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/989/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://fd99030cc9077842d614193d352316e4ef81dada-chi-pr.enterprise.ctl.io" target="_blank"> https://fd99030cc9077842d614193d352316e4ef81dada-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/990/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cebbef904bc7de79796b8cc48e2cab270facf358-chi-pr.enterprise.ctl.io" target="_blank"> https://cebbef904bc7de79796b8cc48e2cab270facf358-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/992/ for more details.

File: src/website/views/components/drawer/_examples.pug [hunk 1]
```diff
@@ -444,14 +720,17 @@ p.-text
...
+      <!-- Trigger -->
       <chi-button id="drawer-trigger" type="icon">
-        <chi-icon icon="menu"></chi-icon>
+        <chi-icon icon="menu" size="sm--2"></chi-icon>
       </chi-button>
 
+      <!-- Drawer -->
       <chi-drawer id="interactive-drawer" position="right" backdrop>
         <div class="-px--2 -text">Drawer content here</div>
       </div>
```
Comment (mattnickles) [inline src/website/views/components/drawer/_examples.pug:731]:
```suggestion
      </chi-drawer>
```

File: src/website/views/components/drawer/_examples.pug [hunk 2]
```diff
@@ -217,19 +354,43 @@ h3 Titled
         a(href='#html-titled') HTML Blueprint
   .chi-tabs-panel.-active#webcomponent-titled
     :code(lang="html")
-      <chi-drawer position="left" title="Drawer title here" backdrop active>
+      <!-- Trigger -->
```
Comment (mattnickles) [inline src/website/views/components/drawer/_examples.pug:357]: Behavior is slightly inconsistent in all examples. The HTML Blueprint code dismisses the drawer when the backdrop is clicked. The Web Component code does not.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://5ca0ff6edc4c4d6c0a6dd030d443067eef22534d-chi-pr.enterprise.ctl.io" target="_blank"> https://5ca0ff6edc4c4d6c0a6dd030d443067eef22534d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/995/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cf1d31b747fb548142249203eed6ea030f3b298c-chi-pr.enterprise.ctl.io" target="_blank"> https://cf1d31b747fb548142249203eed6ea030f3b298c-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/996/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-ygkszj, https://ops.elasticbox.com/services/instances/i-cryzp5, https://ops.elasticbox.com/services/instances/i-zyya8d, https://ops.elasticbox.com/services/instances/i-a9xpp2

**Pull request merged**